gallery/src/components/NxSmallTag/NxSmallTagInTableExample.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxSmallTag, NxTable } from '@sonatype/react-shared-components';
+
+function NxSmallTagInTableExample() {
```

Comment:
What was the motivation for having this particular example? Does it demonstrate anything meaningful aside from the alignment between an `NxSmallTag` and an adjacent `<span>`?

Reply:
It was demonstrated in the mockups, thinking it wouldn't hurt to have it in the gallery and have the look and feel of a small tag in the table. But if you prefer removing this example, I am fine with it.

Reply:
Removed here d6ee65a1f24a7c3e96a420490e625b4bc9ddb5be